crypto/include/internal/asn1_int.h

```diff
@@ -58,6 +58,9 @@ struct evp_pkey_asn1_method_st {
     int (*pkey_check) (const EVP_PKEY *pk);
     int (*pkey_public_check) (const EVP_PKEY *pk);
     int (*pkey_param_check) (const EVP_PKEY *pk);
+    /* Get/set raw private/public key data */
+    int (*set_priv_key) (EVP_PKEY *pk, const unsigned char *priv, size_t len);
+    int (*set_pub_key) (EVP_PKEY *pk, const unsigned char *pub, size_t len);
```

Comment:
You should add functions to set these, in `crypto/asn1/ameth_lib.c`, and document them in `doc/man3/EVP_PKEY_ASN1_METHOD.pod`

Reply:
Good point. Done.